src/edu/psu/sweng500/team8/play/GameSession.java

```diff
@@ -106,12 +123,23 @@ public void doRedo() {
 	public void enterNumber(CellCoordinates cellCoordinates, int number) {
 		
 		Cell currentCell = board.getCell(cellCoordinates.getRowIndex(), cellCoordinates.getColumnIndex());
-		SudokuAction sudokuAction = new SudokuAction(cellCoordinates, currentCell.getNumber(), number);
 		
-		/* TODO Actual Board Update to be implemented */
+		/* TODO - come back after PencilMarkMaanger is implement
+		 * current assumption is that PecilMarkManager will be a snapshot of the current pencil mark status 

```

Comment:
PencilMarkManager being a singleton and it controlling updates is fine.
So if Cell.pencilMarks is used to keep track of pencil mark per cell, is PencilMarkManager going to iterate through each cell to wipeout when a number is entered?
Regardless, I will update the code to get a snapshot of the board's pencil mark using Cell.pencilMarks.